paasta_tools/paasta_cli/cmds/status.py

```diff
@@ -200,9 +213,20 @@ def paasta_status(args):
         cluster_filter = args.clusters.split(",")
     else:
         cluster_filter = None
+    if args.instances is not None:
+        instance_filter = args.instances.split(",")
```

Comment:
IMO ``instance_filter`` is a misnomer - filters are generally functions rather than lists; I'd expect 'desired_instances' or something.

Saying that, I see that we use ``cluster_filter`` and ``bogus_filters`` elsewhere, so I guess you're just following prior art here.